docs/config.md

```diff
@@ -124,12 +124,6 @@ Address of the DApp's contract.
 
 * **Type:** `string`
 
-## `CARTESI_CONTRACTS_ICONSENSUS_ADDRESS`
```

Comment:
Why have we removed this?

Reply:
It's no longer in use since we get the iconsensus address from the application on the repository now